docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -270,11 +274,7 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
       rtl: showRtl,
     }
 
-    return (
-      <Provider theme={newTheme}>
-        <SourceRender.Consumer>{({ element }) => element}</SourceRender.Consumer>
-      </Provider>
-    )
+    return <Provider theme={newTheme}>{element}</Provider>
```

Comment:
this seems to be another problem - seems that `theme` was intended to be used here

Reply:
my bad here, not noticed that some lines are hidden